lib/StaticNarrative/exporter/app_processor.py

```diff
@@ -54,7 +58,9 @@ def process(
         kb_info["output"] = {
             "widget": exec_state.get("outputWidgetInfo", {}),
             "result": exec_result,
-            "report": build_report_view_data(self.host, self.ws_client, exec_result),
+            "report": build_report_view_data(
+                self.ws_client, self.config.indexed_data, self.host, exec_result
+            ),
```

Comment:
some of the data needed to build the report view is already available, so let's use it!